.appveyor.yml

```diff
@@ -12,7 +12,8 @@ build: off
 
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
-  - npm install
+  - npm install --global npm@5.8.0
```

Comment:
I'd prefer to do `npm i -g npm` instead of locking to a version so that we always use the latest version